migrations/sql/V2023.08.21.1254__AddPercentGaviSupportToCoverage.sql

```diff
@@ -0,0 +1,5 @@
+-- To carry out new impact calculations we need to indicate
+-- percentage of GAVI support in a given vaccination activity.
+ALTER TABLE coverage
+    ADD COLUMN percent_gavi_support DECIMAL DEFAULT NULL
```

Comment:
```suggestion
    ADD COLUMN percent_gavi_support DOUBLE PRECISION DEFAULT NULL
```

The "decimal" type has much worse performance. I see we still have a bunch of those around though. It's not super obvious which is which typically!

https://www.postgresql.org/docs/current/datatype-numeric.html